src/workflows/transfers.ts

```diff
@@ -69,15 +74,29 @@ export async function transfersWorkflow({
 }
 
 export async function batchTransfersWorkflow({
-  l1Signer,
-  l2Signer,
+  ethSigner,
+  starkSigner,
   request,
   transfersApi,
 }: BatchTransfersWorkflowParams): Promise<CreateTransferResponse> {
+  const ethAddress = await ethSigner.getAddress();
+
+  const signableRequests = request.map(x => {
```

Comment:
What is `x`? Can we make this name clearer or strictly type the param?

Reply:
Request is an array of `NftTransferDetails` so it is already strictly typed. I'll rename `x` to `nftTransfer`

Reply:
[fc0cb1f](https://github.com/immutable/imx-core-sdk/pull/185/commits/fc0cb1f5944ec32255b0c3f5077e515af50b3b61)